src/components/badge/index.tsx

```diff
@@ -1,12 +1,13 @@
 import React, { FC } from 'react';
-import { Badge as ChakraBadge } from '@chakra-ui/react';
+import {
+  Badge as ChakraBadge,
+  BadgeProps as ChakraBadgeProps,
+} from '@chakra-ui/react';
 
-export interface BadgeProps {
+export type BadgeProps = Exclude<ChakraBadgeProps, 'children'> & {
```

Comment:
Since we re-map `text` to `children`. I exclude it here to avoid unexpected behaviour